service/lib/dinstaller/software.rb

```diff
@@ -146,9 +144,13 @@ def import_gpg_keys
     end
 
     def add_base_repo
-      base_url = Yast::InstURL.installInf2Url("")
-      base_url = FALLBACK_REPO if base_url.empty?
-      Yast::Pkg.SourceCreateBase(base_url, "/")
+      # TODO properly pass config around. Here it is just for quick testing
+      config = Config.new
```

Comment:
I know it is just temporary code, but I would prefer to inject this configuration file through the constructor. So the manager could read the config (and eventually validate it) and pass it to each "module" (software, language, and so on).

Reply:
OK, that's just what you commented above. But let's change it before merging, please :-)